proto/sifnode/dispensation/v1/query.proto

```diff
@@ -37,4 +39,11 @@ message QueryRecordsByRecipientAddrRequest { string address = 1; }
 message QueryRecordsByRecipientAddrResponse {
   DistributionRecords distribution_records = 1;
   int64 height = 2;
+}
+
+message QueryClaimsByTypeRequest { DistributionType user_claim_type = 1; }
+
+message QueryClaimsResponse {
```

Comment:
Needs to be QueryClaimsByTypeResponse ... shouldn't be reused by other claim queries.

Reply:
Claims response ,is used to respond to QueryClaimsByTypeRequest for now . but in the future will also be used to return QueryClaimsbyAddress , and QueryClaimsByStatus  ,which both return a list of claims